plutus-core/testlib/PlutusIR/Generators/QuickCheck/ShrinkTerms.hs

```diff
@@ -48,10 +46,6 @@ addTmBind (TermBind _ _ (VarDecl _ x a) _) = Map.insert x a
 addTmBind (DatatypeBind _ dat)             = (Map.fromList (matchType dat : constrTypes dat) <>)
 addTmBind _                                = id
 
-shrinkConstant :: DefaultUni (Esc a) -> a -> [Term TyName Name DefaultUni DefaultFun ()]
-shrinkConstant uni x =
-    map (mkConstantOf () uni) $ bring (Proxy @ArbitraryBuiltin) uni $ shrinkBuiltin x
```

Comment:
I somehow managed to integrate shrinking of constants into the term shrinker the wrong way. It's now fixed and we no longer need this definition.

Reply:
Does this exist elsewhere, or are we no longer shrinking constants?

Reply:
We now shrink constants in a better way here:

```haskell
        Constant _ val ->
          shrink val <&> \val'@(Some (ValueOf uni _)) ->
            (mkTyBuiltinOf () uni, Constant () val')
```

The new code directly uses all the fancy constant-shrinking machinery that can change kinds, shrink to `unit` etc.

I.e. I got rid of garbage in favor of doing things how they were supposed to be done from the start.